src/lib.rs

```diff
@@ -66,83 +132,53 @@
 //!                                .short('v')
 //!                                .multiple(true)
 //!                                .help("Sets the level of verbosity"))
-//!                           .subcommand(App::new("test")
+//!                           .subcommand(SubCommand::with_name("test")
```

Comment:
I've been gone for a while, but the original intent was to get rid of `SubCommand` and just move to `App` (this that's actually what `SubCommand`s are under the covers). 

cc @CreepySkeleton @pksunkara

Reply:
Still the same intent. I think the user made a mistake. The README still contains references to `SubCommand` which is probably why he did this. I already updated the README in my local branch, working on other stuff before pushing it.

Reply:
Yeah, right. Fun fact: we still have `SubCommand` in public API:

https://github.com/clap-rs/clap/blob/5b9dbee5db540f44c336c2db54ece444624c30ef/src/parse/matches/subcommand.rs#L4-L30

What's interesting here, even though this is totally not the `SubCommand` from 2.x, it still has the doc comment pretending that it is 😃 😕 😢 

I'm going to make it `pub(crate)`.

Reply:
You can't make it private since it is part of `ArgMatcher`

Reply:
So what? 🤷 It's a `pub(crate)` field, no mentioning in signatures and such... Just an artifact from days of old.